src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/MediaBlobContainerTenantEvents.cs

```diff
@@ -0,0 +1,87 @@
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Environment.Shell.Removing;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Media.Azure.Services;
+
+public class MediaBlobContainerTenantEvents : ModularTenantEvents
+{
+    private readonly MediaBlobStorageOptions _options;
+    private readonly ShellSettings _shellSettings;
+    protected readonly IStringLocalizer S;
```

Comment:
please place protected properties below private ones followed a single new line. No there is no advantage here other than consistency

Reply:
I agree, though I didn't change this; while GitHub doesn't show for some reason, I only did minimal changes in this class and only moved it after adding the ImageSharp services.